homeassistant/components/peco/const.py

```diff
@@ -13,3 +15,15 @@
     "TOTAL",
 ]
 SCAN_INTERVAL = 5
+ATTR_CUSTOMERS_OUT = "customers_out"
+ATTR_PERCENT_CUSTOMERS_OUT = "percent_customers_out"
+ATTR_OUTAGE_COUNT = "outage_count"
+ATTR_CUSTOMERS_SERVED = "customers_served"
+SENSOR_LIST = (
+    SensorEntityDescription(key=ATTR_CUSTOMERS_OUT, name="{} Customers Out"),
+    SensorEntityDescription(
+        key=ATTR_PERCENT_CUSTOMERS_OUT, name="{} Percent Customers Out"
+    ),
+    SensorEntityDescription(key=ATTR_OUTAGE_COUNT, name="{} Outage Count"),
+    SensorEntityDescription(key=ATTR_CUSTOMERS_SERVED, name="{} Customers Served"),
+)
```

Comment:
move this to `sensor.py` as it is only sensor relevant.
Also define units, device_class and state_class where applicable - eg
```py
from homeassistant.const import PERCENTAGE

        native_unit_of_measurement=PERCENTAGE,
        state_class=SensorStateClass.MEASUREMENT,
```
Imho it doesn't need constants for the keys - thats what the SensorEntityDescriptions are for to define.

Reply:
For the percent customers out sensor, PECO does not provide values below 5%. What should I do then?